### ISSUE TITLE
text message sent when accepting a request disappears

### ISSUE THREAD

**Issue by fkleedorfer** [researchstudio-sat/webofneeds#501]
When I get a request and send a text message when I accept it, that message is never displayed anywhere (at least on my side).

maybe related to #500

**Comment by fkleedorfer** [comment]
Update: the original requester saw my text message. 
When I got another message from the original requester, the conversation view updated and also showed the "accept" message, but after my other messages.

**Comment by fkleedorfer** [comment]
Also, the ordering of messages is inconsistent for both users

**Comment by fkleedorfer** [comment]
The behavour has changed: the request message (that I typed when sending the request) is now displayed above the textDescription. Changing the issue title.
![accept-printer](https://cloud.githubusercontent.com/assets/551624/18096816/d66e449e-6edb-11e6-894f-9eb70af8162d.PNG)

**Comment by m0ru** [comment]
There's currently a server-side filter in the code for deep-fetching event-containers, that causes the accept-event to not get loaded. e.g:

```
GET https://localhost:8443/owner/rest/linked-data/?requester=https:%2F%2F192.168.124.53:8443%2Fwon%2Fresource%2Fneed%2F5431319554379530000&uri=https:%2F%2F192.168.124.53:8443%2Fwon%2Fresource%2Fconnection%2Fxzorfkdc8g3thlohpgyr%2Fevents%3Fdeep%3Dtrue

=>

"@graph" : [ {
    "@graph" : [ {
      "@id" : "conn:xzorfkdc8g3thlohpgyr/events",
      "rdfs:member" : [ {
        "@id" : "event:0as57ypiugihsr239hdn"
      }, {
        "@id" : "event:27uh1svdg6cbk37ow3hg"
      }, {
        "@id" : "event:lkp5tzz1bef5vxu80hp6"
      }, {
        "@id" : "event:6gvx1me9yklnboi1pkwx"
      }, {
        "@id" : "event:fzvufws5b0ey6kk1gnwi"
      }, {
        "@id" : "event:0i081ax7b9tnbc97botq"
      }, {
        "@id" : "event:4eqlwcmt7hf5z06crmq8"
      }, {
        "@id" : "event:1t0g0uzuoll3h4zo8g6k"
      }, {
        "@id" : "event:8j54g1nlc1jiuavf11b7"
      }, {
        "@id" : "event:4guruh3s6t4s8qk44jm2"
      }, {
        "@id" : "event:vmrxbkl5vsukq453wty4"
      }, {
        "@id" : "event:brd26ikseu1xg82m8ywu"
      } ]
    } ],
...
```

whereas on <https://192.168.124.53:8443/won/page/connection/xzorfkdc8g3thlohpgyr/events>:

```
<https://192.168.124.53/won/resource/connection/xzorfkdc8g3thlohpgyr/events#data> {
    <https://192.168.124.53:8443/won/resource/connection/xzorfkdc8g3thlohpgyr/events>
            a            won:EventContainer ;
            rdfs:member  event:0as57ypiugihsr239hdn , event:m16azsugg2mgfci0bntm , event:27uh1svdg6cbk37ow3hg , event:lkp5tzz1bef5vxu80hp6 , event:6gvx1me9yklnboi1pkwx , event:fzvufws5b0ey6kk1gnwi , event:cf2hosgvq2i0205kk1l0 , event:0i081ax7b9tnbc97botq , event:tol3e55l4m52asgdz4ws , event:4eqlwcmt7hf5z06crmq8 , event:y1giv2txeqd2jgaizcvm , event:1t0g0uzuoll3h4zo8g6k , event:8j54g1nlc1jiuavf11b7 , event:4guruh3s6t4s8qk44jm2 , event:vmrxbkl5vsukq453wty4 , event:3iyxndgoma4air5ilkuq , event:j6574ohx8yj25fpsiiw1 , event:6873134215601527000 , event:brd26ikseu1xg82m8ywu .
}
```

specifically the `event:6873134215601527000` is missing in the GET-request above

**Comment by m0ru** [comment]
The above is only half of this bug, though. The event apparently also doesn't get added to the rdfstore and client-state when it's sent (no round-trip should be needed).

**Comment by m0ru** [comment]
Hmm, sometimes reloading lists the event. Weird. Need more debugging.

**Comment by m0ru** [comment]
Server loading event containers with deep=true seems to work now. I assume, that maybe earlier this might have been a race condition, and thus a false diagnosis.

**Comment by m0ru** [comment]
Error recovery is partially implemented for the rest see #917
